client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
```

Comment:
Forgot to touch on this :smile:

> Also wondering if you're thinking the whole config object would be in the API request, or just the id?

I'm not sure whats best here. I feel like the `id` definitely doesn't belong here but I feel like you could make good arguments for both keeping and removing the default display options.